### DISCUSSION TITLE
Lodestar Standup 2024-19 (May 21, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6803] [Standups]
### Meeting Info
[May 21, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/may-21-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - Interop Retro
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by twoeths** [comment]
need to summarize how many blocks proposed in v1.18.1 with `disableFloodPublish` flag. If there are a good amount, may turn it on by default for v1.19.0

**Comment by philknows** [comment]
Some discussion here about docs layout. Feel free to come and suggest anything based on these comments: https://github.com/ChainSafe/lodestar/issues/6550#issuecomment-2117593526

**Comment by philknows** [comment]
Closing for #6809
